facia/app/controllers/FaciaController.scala

```diff
@@ -25,6 +25,14 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
 
   val frontJsonFapi: FrontJsonFapi
 
+  private def getEditionFromString(edition: String) = {
+    val editionToFilterBy = edition match {
+      case "international" => "int"
+      case _ => edition
+    }
+    Edition.all.find(_.id.toLowerCase() == editionToFilterBy).getOrElse(Edition.all.head)
+  }
+
```

Comment:
I am happy to reevaluate passing the edition as a parameter once we can reliably get it from the request.